src/app/clusters/color-control-server/color-control-server.h

```diff
@@ -20,14 +20,17 @@
 #include <app-common/zap-generated/cluster-objects.h>
 #include <app/CommandHandler.h>
 #include <app/ConcreteCommandPath.h>
-#include <app/clusters/scenes-server/SceneTable.h>
 #include <app/util/af-types.h>
 #include <app/util/attribute-storage.h>
 #include <app/util/basic-types.h>
 #include <app/util/config.h>
 #include <platform/CHIPDeviceConfig.h>
 #include <protocols/interaction_model/StatusCode.h>
 
+#ifdef MATTER_DM_PLUGIN_SCENES_MANAGEMENT
```

Comment:
This is extremely fragile, since it relies on util/config.h being included before this part, but absolutely nothing checks that it is.

We need to figure out a better approach here.  Followup for that, please....  Possible options:

1. Use 1/0 vs defined/not and change the codegen accordingly, then use `#if`, which will fail if it's not defined at all.
2. static_assert something that is defined in config.h any time we rely on it being included.
3. Others?

Reply:
My preference leans towards solution 1. since it doesn't cost much to have every clusters defined by default to 0 and it won't be too different from what we are currently doing.

Opened a followup here:
https://github.com/project-chip/connectedhomeip/issues/33659